tools/setup/emoji/build_emoji

```diff
@@ -239,6 +252,30 @@ def dump_emojis(cache_path):
             os.path.join(cache_emoji, fn)
         )
 
+    run(['cp', os.path.join(EMOJI_DATA_PATH, 'sheet_google.png'), cache_path], shell=True)
+    emoji_classes = []
+    emoji_class = ".emoji_%(codepoint)s"
+    emoji_positions = ""
+    emoji_pos_info = """
+        .emoji_%(codepoint)s {
+            background-position: -%(pos_x)spx -%(pos_y)spx;
+        }
+    """
+    for emoji in emoji_data:
+        if emoji["has_img_google"]:
+            emoji_classes.append(emoji_class % {"codepoint": emoji["unified"], })
+            emoji_positions += emoji_pos_info % {'codepoint': emoji['unified'],
+                                                 'pos_x': emoji["sheet_x"] * 20,
+                                                 'pos_y': emoji["sheet_y"] * 20,
```

Comment:
Let's use percent, not pixels, for this; that is necessary for resolving the emoji scaling performance issue.

(If you play with Slack's CSS for their emoji, you can see an example of how this works).